stack.go

```diff
@@ -43,17 +41,6 @@ func BuildStack(skip int) Stack {
 	return stack
 }
 
-// Fingerprint builds a string that uniquely identifies a Rollbar item using
-// the full stacktrace. The fingerprint is used to ensure (to a reasonable
-// degree) that items are coalesced by Rollbar in a smart way.
-func (s Stack) Fingerprint() string {
```

Comment:
IMO. I would leave this in so that people can get the existing behavior by doing:

```golang
stack := rollbar.BuildStack(2)
ErrorWithStack(rollbar.ERR, err, stack, rollbar.Field{"fingerprint", stack.Fingerprint())
```